### ISSUE TITLE
[Question] test.skip inside a test – is that a correct way to skip the test?

### ISSUE THREAD

**Issue by Diokuz** [microsoft/playwright#16010]
We have some strange behavior:

```
✘  [firefox] › packages/forms/dolyame/__playwright__/fields/email/emailValidation.test.ts:93:1 › *** (122ms)
-  [firefox] › packages/forms/dolyame/__playwright__/fields/email/emailValidation.test.ts:93:1 › *** (retry #1)

...

 1) [firefox] › packages/forms/dolyame/__playwright__/fields/email/emailValidation.test.ts:93:1 › *** 
    Error: expect(received).toEqual(expected) // deep equality
    Expected: 1
    Received: 0
    Call Log:
    - Timeout 5000ms exceeded while waiting on the predicate
  1 failed
    [firefox] › packages/forms/dolyame/__playwright__/fields/email/emailValidation.test.ts:93:1 › ***
  37 skipped
  37 passed (29s)
```

Test is actually __skipped__

```
test('***', async ({ storybook, dolyame, page }) => {
  allure.description([
    `Human readable text for manual QA`,
  ].join('\n'))

  test.skip(true, 'Still working on it');

  await storybook.goto('Primary')
  await dolyame.fillField('email', 'test@test.com')

  ...
})
```

We followed this doc to skip test and be able to set allure.description at the same time https://playwright.dev/docs/test-annotations#skip-a-test

Sometimes retry leads to ✓, sometimes to ✘.

Why test fails if it skipped?

Is there a better way to skip test and set description for allure report?

Playwright version is 1.24.1, docker version is 1.24.0, allure-report version is 2.0.0-beta.18

PS: Chrome and webkit both are ok, just show that test as skipped.

**Comment by Diokuz** [comment]
One more thing: we have `retries: 7`, and there is just one retry.

**Comment by pavelfeldman** [comment]
It is Ok to skip the test this way, maybe your fixtures are flaky and are throwing sometimes, so it never comes to the test.skip().

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.

**Comment by Diokuz** [comment]
Well, the trick is, I dont have any assertion in my fixtures.
